target/games/controller.js

```diff
@@ -43,42 +42,52 @@ let GameController = class GameController {
             throw new routing_controllers_1.BadRequestError(`Game is already started`);
         game.status = 'started';
         await game.save();
-        const player = await entities_1.Player.create({
+        await entities_1.Player.create({
             game,
             user,
             symbol: '2'
         }).save();
-        const gameToSend = await entities_1.Game.findOneById(game.id);
-        if (!gameToSend)
-            throw new routing_controllers_1.BadRequestError(`Game does not exist`);
         index_1.io.emit('action', {
             type: 'UPDATE_GAME',
-            payload: Object.assign({}, gameToSend, { board1: gameLogic_1.getGuessBoard(gameToSend.board1) })
+            payload: gameLogic_1.gameToSend(game)
         });
-        return player;
+        return { board: game.board2, guessBoard: game.board1 };
     }
     async getGame(user, id) {
         const game = await entities_1.Game.findOneById(id);
         if (!game)
             throw new routing_controllers_1.BadRequestError(`Game does not exist`);
+        const toSend = gameLogic_1.gameToSend(game);
         const player = await entities_1.Player.findOne({ user, game });
         if (!player)
-            return Object.assign({}, game, { board1: gameLogic_1.getGuessBoard(game.board1), board2: gameLogic_1.getGuessBoard(game.board2) });
```

Comment:
gameLogic_1 iiiiis? something from our game logic haha

Reply:
Well, this is just the way typescript compiled it I think. Oh, in fact we can also put js files in .gitignore like our last project

Reply:
oh yes, I think that would be the best